src/plugins/intel_gna/src/transformations/insert_identity_layer.hpp

```diff
@@ -68,6 +68,73 @@ class IdentityCandidatesCleanup : public ov::pass::ModelPass {
     bool run_on_model(const std::shared_ptr<ov::Model>& m) override;
 };
 
+/**
+ * @brief Inserts identity for precision agnostic (or FQ) concat inputs
+ * Example model:
+ *
+ *                      Parameter
+ *                          |
+ *   Functional      Prec-Agnostic or FQ
+ *              \       /
+ *               Concat
+ *                  |
+ *               Result
+ *
+ * For the above model, during scale factors propagation, when left Concat input
+ * will be selected as a scale factors source, then algorithm will not be able to
+ * apply the scale factors to the right Concat input and will throw exception.
+ * InsertIdentityForPrecAgnosticConcatInput pass adds Identity layer, which
+ * is capable of storing scale factors, so the scale factors propagation can proceed:
+ *
+ *                    Parameter
+ *                        |
+ *                  Prec-Agnostic or FQ
+ *                        |
+ *      Functional     Identity
```

Comment:
1.what in case if there will be more than two inputs to concat?
2. what in case second input will be functional and first Prec-agnostic?

Reply:
Updated description.